### DISCUSSION TITLE
WHY DON'T YOU FIX THE BUG ACCORDING TO THIS PROBLEM?

### DISCUSSION THREAD

**Discussion by L-J-Yinph** [saicaca/fuwari#255] [General]
Suddenly find that the screen is too small, and the TOC will disappear...... WHY DON'T YOU FIX THE BUG ACCORDING TO THIS PROBLEM?

**Comment by saicaca** [comment]
This is the expected behavior. The TOC will hide when there is not enough space. I do plan to add a button to open the TOC for smaller screens, but it will take some time.
